compatibility.json

```diff
@@ -0,0 +1,18 @@
+{
+  "cli": {
+    "latest": "0.14.2",
+    "compatibility": [
+      {
+        "range": ">0.0.1"
```

Comment:
I removed the `compatibility` field because I couldn't figure out how to validate those. I think it makes more sense to only pass valid ranges in this array and whatever does not satisfy those is invalid, WDYT? Could maybe even be simplified to just strings instead of objects?